idea_town/frontend/static-src/app/views/footer.js

```diff
@@ -0,0 +1,6 @@
+import BaseView from './base-view';
+import template from '../templates/footer';
```

Comment:
hey @lmorchard...good catch. i had originally called this file footer.js then changed it to footer-view.js to better match the naming convention for views and pages. i guess i forgot to git remove this one